test_script.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+####### Run test cases for pyscript.py script #######
+### Remove Old Logs
+rm -rf data.log
+
+### Test Case # 1
+echo 'aaabccccdd' | python3 pyscript.py >> data.log
+
+### Test Case # 2
+echo 'aaaaaffffffffffc' | python3 pyscript.py >> data.log
+
+### Test Case # 3
+echo 'abcd' | python3 pyscript.py >> data.log
+
+### Test Case # 4
+echo 'ccceee12eccceee' | python3 pyscript.py >> data.log
+
+### Test Case # 5
+echo 'effeac01cb65c' | python3 pyscript.py >> data.log
+
+### Print Expected Result
+echo '----------Expected Result----------'
+VAR1=`cat expected_result`
+echo $VAR1
```

Comment:
Rather than use an external file you could:
1. Update this script to accept the file contents via STDIN. That makes the script more robust b/c it doesn't have to know where that file is
    - e.g. `$ ./test_script.sh < ./path/2/file.txt`
1. Update it to take N arguments. 
    - e.g. `$ ./test_script.sh effeac01cb65c ccceee12eccceee aaaaaffffffffffc`
1. Include the file contents right in this script
```bash
```suggestion
echo '----------Expected Result----------'

VAR1=$(cat <<CHANGEME
a3bc4d2
a5f10c
abcd
c3e4c3e3
ef2eac2bc
CHANGEME
)

echo "$VAR1"
```